docs/repo/crates/discv4.md

```diff
@@ -183,7 +183,7 @@ impl Discv4Service {
 }
 ```
 
-First, new channels are initialized to handle incoming and outgoing traffic related to node discovery. New tasks are then spawned to handle the `receive_loop()` and `send_loop()` functions, which perpetually send incoming disc4 traffic to the `ingress_rx` and outgoing discv4 traffic to the `egress_rx`. Following this, a [`KBucketsTable`]() is initialized to keep track of the node's neighbors throughout the network. If you are unfamiliar with k-buckets, feel free to [follow this link to learn more](https://en.wikipedia.org/wiki/Kademlia#Fixed-size_routing_tables). Following this, the `self_lookup_interval`, `ping_interval` and `evict_expired_requests_interval` are initialized which determine the intervals between self lookups, outgoing ping packets and the time elapsed to remove inactive neighbors from the node's records.
+First, new channels are initialized to handle incoming and outgoing traffic related to node discovery. New tasks are then spawned to handle the `receive_loop()` and `send_loop()` functions, which perpetually send incoming disc4 traffic to the `ingress_rx` and outgoing discv4 traffic to the `egress_rx`. Following this, a [`KBucketsTable`](https://docs.rs/discv5/0.1.0/discv5/kbucket/struct.KBucketsTable.html) is initialized to keep track of the node's neighbors throughout the network. If you are unfamiliar with k-buckets, feel free to [follow this link to learn more](https://en.wikipedia.org/wiki/Kademlia#Fixed-size_routing_tables). Following this, the `self_lookup_interval`, `ping_interval` and `evict_expired_requests_interval` are initialized which determines the interval used for periodically querying the network for new neighboring nodes that have a similar node Id to our node, outgoing ping packets and the time elapsed to remove inactive neighbors from the node's records.
```

Comment:
I'd be careful about saying "similar node ID to our node", may give the wrong impression. Maybe something like "node ID with low "distance" relative to ours", and link to https://github.com/ethereum/devp2p/blob/master/discv4.md#node-identities for an explanation of "distance"

Reply:
Good idea, I'll change this to your suggestion.